ethereum/test/Treasury.t.sol

```diff
@@ -33,7 +33,8 @@ contract TreasuryTest is TestHelper {
         treasury.deposit(address(erc20_token), amount, VARA_ADDRESS_3);
     }
 
-    function test_withdraw() public {
+    // Test skipped, to enable it remove the skip_ prefix
```

Comment:
Please also add `TODO` to find this comment later

Reply:
Fixed